jobs/management/commands/populate_database.py

```diff
@@ -0,0 +1,152 @@
+import random
+
+import os
+
+from django.core.management.base import NoArgsCommand
+
+from jobs.models import Company, Job, Meetup
+
+
+def read_lines():
+    """Reads random lines from a file to generate description."""
+    name = 'jobs/management/commands/description.txt'
+    if os.path.isfile(name):
+        with open(name, 'r') as f:
+            all_lines = f.read().splitlines()
+            for i in range(0, 4):
+                selected_lines = random.choice(all_lines)
+                description = ''.join(selected_lines)
+    else:
+        description = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+    return description
```

Comment:
I really like how you did this, but unfortunately it means that we get Integrity Errors from the manage command.  If you rerun the command, the get_or_create in add_meetup and add_job always tries to create, because the description is randomly generated and always different - so it tries to create an object with a title/company (for jobs) or title/city (for meetups) which violates the unique together constraint on the model.